### ISSUE TITLE
html2text throws ValueError on invalid unicode entities

### ISSUE THREAD

**Issue by nikolas** [Alir3z4/html2text#74]
Using html2text 2015.4.14, python 2.7.9 on Ubuntu 15.06, x86_64:
```
ipdb> p html2text.html2text('B&#3291685;r')
*** ValueError: ValueError('unichr() arg not in range(0x110000) (wide Python build)',)
ipdb> p html2text.html2text(u'B&#3291685;r')
*** ValueError: ValueError('unichr() arg not in range(0x110000) (wide Python build)',)
```
Possibly addressed by PR #60?

**Comment by theSage21** [comment]
@nikolas What should this unicode entity be translating into?

**Comment by nikolas** [comment]
probably an empty string since that unicode entity doesn't exist

**Comment by Alir3z4** [comment]
Fixed in #79
